transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -34,6 +34,8 @@
 import java.net.SocketAddress;
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
 
+import static io.netty.channel.ChannelHandlerMask.*;
```

Comment:
No wildcard imports, right?

Reply:
You are right we usually dont do these... Let me change it

Reply:
@rkapsi done